src/containers/Review/ReviewPage.tsx

```diff
@@ -61,9 +61,7 @@ const ReviewPage: React.FC<{
 
   const { isSectionActive, toggleSection } = useQuerySectionActivation({
     defaultActiveSectionCodes: [],
-    allSections: (reviewStructure?.sortedSections as SectionState[]).map(
-      (section) => section.details.code
-    ),
+    allSections: Object.keys(fullApplicationStructure.sections).map((section) => section),
```

Comment:
That was what broke the `ReviewPage` : ( I think at that point there wasn't any `sortedSection` on the structure yet, since it hasn't received back from the hook. That's what I found out while debugging

Reply:
Oh, sorry I didn't read this fully! You are completelly right. I'll change this now

Reply:
Done